packages/fields/src/actions/trash-post.tsx

```diff
@@ -27,7 +27,7 @@ const trashPost: Action< PostWithPermissions > = {
 	isPrimary: true,
 	icon: trash,
 	isEligible( item ) {
-		if ( isTemplateOrTemplatePart( item ) || item.type === 'wp_block' ) {
```

Comment:
We should probably keep this for "template parts" no?

Reply:
For now, yes.